spec/src/lib.rs

```diff
@@ -87,48 +87,43 @@ pub struct IssuedCell {
 }
 
 #[derive(Debug)]
-pub struct FileNotFoundError;
-
-impl FileNotFoundError {
-    fn boxed() -> Box<Self> {
-        Box::new(FileNotFoundError)
-    }
+pub enum SpecLoadError {
+    FileNotFound,
+    SpecNameNotAllowed(String),
```

Comment:
If change the commandline option `--spec` to `--chain`, then may adjust the error messages.